include/arch/arm/arch/64/mode/object/structures.bf

```diff
@@ -176,9 +148,7 @@ tagged_union cap capType {
     -- 5-bit tag arch caps
     tag frame_cap                   1
     tag page_table_cap              3
-    tag page_directory_cap          5
-    tag page_upper_directory_cap    7
-    tag page_global_directory_cap   9
+    tag vspace_cap                  9
```

Comment:
I don't think there's a requirement that they're consecutive (maybe the proofs have some requirement here?), but there's the seL4 debug call  that user level can use to identify a captype from the cap number, so leaving the numbers the same avoids some user level breakage here since we can.